### ISSUE TITLE
Java using modules (-p and -m options)

### ISSUE THREAD

**Issue by pzw** [tofi86/universalJavaApplicationStub#83]
I'm submitting a…

- [ ] bug report
- [x] feature request
- [ ] other

Do you plan to have a way using Java 11/14 with module-path instead classpath ?
1) a way to give modules path and main module to JarBundler ANT Task
2) updating universalJavaApplicationStub to launch java with -p and -m options

**Comment by tofi86** [comment]
Good catch. I haven't worked with the new Java module system yet, so this was indeed a bit out of scope.

You also pointed out, that this requires an updated JarBundler ANT task so I would need to synchronize development with the other project.

Not sure yet, when to find time for this at the moment...

**Comment by pzw** [comment]
OK, thank you for your answer.

I use a modified version of your universalJavaApplicationStub. It use two additionnal entries in java dictionnary of Info.plist : ModulePath and MainModule. These entries are manually added to Info.plist, because I have not worked on ANT task to produce this file automatically). They are used to launch java in "modular mode" if not class path is defined.

Best regards
